guides/common/modules/con_making-open-source-more-inclusive.adoc

```diff
@@ -0,0 +1,6 @@
+[id="making-open-source-more-inclusive_{context}"]
+= Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties.
+Because of the enormity of this endeavor, these changes are being updated gradually and where possible.
+For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
```

Comment:
```suggestion
For more details, see https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
```

Reply:
I was going to ask about the link formatting at some point. The [RH Asciidoc quick ref](https://redhat-documentation.github.io/asciidoc-markup-conventions/), [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#external-links), and [openshift-doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites) require the link macro instead of the URL macro, even for ordinary URLs. I'm not sure why. We do not have many style guides that cover Asciidoc formatting. 

Is there a good reason not to use the link macro?

Reply:
@maximiliankolb 
Is there a reason not to use the link macro?